lib/crypto/OlmDevice.js

```diff
@@ -514,21 +514,23 @@ OlmDevice.prototype.getOutboundGroupSessionKey = function(sessionId) {
  * store an InboundGroupSession in the session store
  *
  * @param {string} roomId
- * @param {string} senderKey
+ * @param {string} senderCurve25519Key
  * @param {string} sessionId
  * @param {Olm.InboundGroupSession} session
+ * @param {object} keysClaimed Other keys the sender claims.
```

Comment:
1. It's a shame that this doesn't specify the format of the object. But that doesn't matter too much, because it's a private method.

2. It does't appear to be set by either `addInboundGroupSession` or `decryptGroupMessage`.